### DISCUSSION TITLE
[BUG REPORT]GitHub Classoom Repository assigned but invalid

### DISCUSSION THREAD

**Discussion by CodrinCristea-si** [community/community#13423] [Projects and Issues]
Hello, last week when the servers were down, I've accessed GitHub Classroom and I've assigned a repository for a project, the link was created, but every time I click it gives me the 404 Error. I tried to unlink that repository from my account but nothing worked. I hope that original repository can be deleted for that link so I can reassign a new one.
The link is https://github.com/LauraDiosan-CS/ai-lab3-CodrinCristea-si
Below I've attached some screenshots with the problem.
![bug1](https://user-images.githubusercontent.com/83692101/159930723-468bece7-47d6-41a5-a94f-ad83b505d517.png)
![bug2](https://user-images.githubusercontent.com/83692101/159930729-fca0211c-7ce3-4b96-a51c-2834c27d7be3.png)

**Comment by Wonderland2333** [comment]
Have you solved it? I encountered the same problem.

**Comment by edhends** [comment]
Hi @CodrinCristea-si  you can report bug here! https://support.github.com/contact/bug-report
